spark/src/main/scala/org/apache/comet/CometExecIterator.scala

```diff
@@ -87,9 +87,9 @@ class CometExecIterator(
       CometSparkSessionExtensions.getCometMemoryOverhead(conf)
     }
 
-    // serialize Spark conf in protobuf format
+    // serialize Comet configs in protobuf format
```

Comment:
```suggestion
    // serialize Comet related Spark configs in protobuf format
```

Reply:
I'm also thinking don't we need usual spark params on native side? like `spark.sql.ansi.enabled`